### ISSUE TITLE
Possable bug in lib_navigation

### ISSUE THREAD

**Issue by nuggreat** [KSP-KOS/KSLib#123]
The user EberKain from the kOS discord reported a problem with `lib_navigation` where line [248](https://github.com/KSP-KOS/KSLib/blob/6b08a973782fb524458bef603287a58bdcf44959/library/lib_navigation.ks#L248)  `local head is arcsin(cos(inclination) / cos(ship:latitude)).` was causing there script to crash with "Tried push a NaN onto the stack" error.

I think this might be caused by a physics tick falling after the check for if the target inclination is lower than the current latitude of the ship.  This might be causing a problem because both the check and the line throwing the error reference `SHIP:LATITUDE` directly.  If the tick does indeed fall between them is is possible that `SHIP:LATITUDE` might be smaller than what it was during the check and crash mitigation step taken can not prevent the crash.  The simple solution here would be to simply store the `SHIP:LATITUDE` in a variable and use that for all checks and calculations thus fixing the issue.

specifically this code 
```
    if abs(inclination) < abs(ship:latitude) {
        set inclination to ship:latitude.
    }

    local head is arcsin(cos(inclination) / cos(ship:latitude)).
```
might need to be changed to this
```
    local localLatitude is ship:latitude.
    if abs(inclination) < abs(localLatitude) {
        set inclination to localLatitude.
    }

    local head is arcsin(cos(inclination) / cos(localLatitude)).
```

I have recommended the above fix to the person who had the problem and am waiting to hear back from there as to if this resolves the error or not.

**Comment by nuggreat** [comment]
The person who had the issue got back to me and my proposed fix does appear to have worked as a fix for the problem

**Comment by scimas** [comment]
Fixed by #124
